Source/NSString+HYPFormula.m

```diff
@@ -31,6 +31,11 @@ - (NSString *)hyp_processValuesDictionary:(NSDictionary *)valuesDictionary;
             NSString *stringValue = (NSString *)value;
             if (!stringValue || stringValue.length == 0) {
                 value = (isNumberFormula) ? @"0" : @"";
+            } else if ([stringValue containsString:@"%"]) {
```

Comment:
I think you could do something like

```objc
NSString *stringValue = (NSString *)value;
NSRange percentRange = [stringValue rangeOfString:@"%"];
if (!stringValue || stringValue.length == 0) {
    value = (isNumberFormula) ? @"0" : @"";
} else if (percentRange.location != NSNotFound) {
    NSString *realStringValue = [stringValue substringToIndex:percentRange.location];
    NSNumber *realValue = [NSNumber numberWithFloat:([realStringValue floatValue] / 100)];
    value = [realValue stringValue];
}
```

That would only look at the string once instead of doing both `containsString` and then finding the range.